src/test/java/carsDB/CarMapperTest.java

```diff
@@ -34,12 +36,17 @@ public void stringToCar() throws Exception {
                 carMapper.stringToCar(carString));
         String carString2 = "XXX/Fiat/126P/1994/BDG 5698";
         try {
-        assertEquals(new Car(0, "Fiat","126P", "1994", "BDG 5698"),
-                carMapper.stringToCar(carString2));
-    } catch (NumberFormatException nfe){
+            assertEquals(new Car(0, "Fiat", "126P", "1994", "BDG 5698"),
+                    carMapper.stringToCar(carString2));
+        } catch (NumberFormatException nfe) {
             String idString = carString2.substring(0, carString2.indexOf("/"));
             System.out.println(idString + " cannot be converted to ID");
-
+        }
+        String carString3 = "8/Rolls/Ghost/2014";
+        try {
+            assertEquals(null, carMapper.stringToCar(carString3));
+        } catch (CarMapperException cme){
```

Comment:
In JUnit we should not use sout, instead exceptions should be tested the way described in this link:
https://stackoverflow.com/questions/156503/how-do-you-assert-that-a-certain-exception-is-thrown-in-junit-4-tests

For exception handling, another (new) test method should be created (e.g. stringToCarExceptionExpected) - one unit test method should test one aspect of the method being tested (e.g, proper data test method, wrong data test method, exception-throwing data test method, etc.)